keyboards/sck/neiso/info.json

```
@@ -0,0 +1,12 @@
+{
+    "keyboard_name": "NEISO",
+    "url": "",
+    "maintainer": "jrfhoutx",
+    "width": 9.25,
+    "height": 14.25,
+    "layouts": {
+        "LAYOUT": {
+            "layout": [{"x":0.25, "y":0, "w":2, "h":1.25}, {"x":2.75, "y":0, "w":1.25, "h":2}, {"x":1.5, "y":1.5}, {"x":0, "y":1.75, "w":1.25, "h":2}, {"x":1.75, "y":2.75, "w":2, "h":1.25}]
```

I messed up the "pinwheel" effect on my previous comment. Moving the second key down 0.25u.

```suggestion
            "layout": [{"x":0.25, "y":0, "w":2, "h":1.25}, {"x":2.75, "y":0.25, "w":1.25, "h":2}, {"x":1.5, "y":1.5}, {"x":0, "y":1.75, "w":1.25, "h":2}, {"x":1.75, "y":2.75, "w":2, "h":1.25}]
```